engine/classes/ElggEntity.php

```diff
@@ -274,6 +274,12 @@ public function __get($name) {
 			return 2;
 		}
 
+		if (in_array($name, array('thumbnail', 'smallthumb', 'largethumb'))) {
```

Comment:
Is there a way to deprecate this without generating a ton of warning when we access these values?